DataFormats/Detectors/TRD/include/DataFormatsTRD/HelperMethods.h

```diff
@@ -156,6 +156,20 @@ struct HelperMethods {
     int sector = hcid / constants::NHCPERSEC;
     return getORIinSuperModule(hcid) + constants::NHCPERSEC * sector;
   }
+
+  static int getMCMId(int sector, int stack, int layer, int rob, int mcm)
+  {
+    // return the mcmid of the indexed mcm
+    int mcmid = (sector * constants::NSTACK * constants::NLAYER + stack * constants::NLAYER + layer) * constants::NROBC1 * constants::NMCMROB + rob * constants::NMCMROB + mcm;
```

Comment:
We already have this nice getDetector function here, why not use it?
```suggestion
    int mcmid = getDetector(sector, stack, layer) * constants::NROBC1 * constants::NMCMROB + rob * constants::NMCMROB + mcm;
```

Reply:
forgot about it

Reply:
In this definition, I do not see space for the board merger and half-chamber merger MCMs. While they don't process signals, their settings can indicate if certain MCMs are excluded from the readout tree.

Reply:
Took a while to verify that, but now that I cant find them, it seems rather obvious why :-( they dont produce digit data and hence of course not in the data stream.